network-api/networkapi/wagtailpages/pagemodels/products.py

```diff
@@ -295,6 +310,45 @@ class ProductPage(FoundationMetadataPageMixin, Page):
         blank=True,
     )
 
+    # Un-editable voting fields. Don't add these to the content_panels.
+    current_total = models.IntegerField(default=0)
+    current_votecount = models.IntegerField(default=0)
+    vote_bin = models.OneToOneField(
+        ProductPageVoteBin,
+        on_delete=models.SET_NULL,
+        null=True,
+        blank=True,
+        related_name='vote_bin',
+    )
+
+    @property
+    def average_vote_count(self):
+        try:
+            total = self.current_votecount / self.current_total
```

Comment:
@pomax I'm wondering if these two properties should be named better. I feel like they're a bit ambiguous at the moment.

Reply:
remember that we're not so much counting "votes" as "creepiness score", so:

- `def average_vote_count` -> `def creepiness`
- `total` -> `average`
- `current_votecount` -> `current_tally`
- `current_total` -> `current_vote_count`